### DISCUSSION TITLE
v8.2 beta minor update for Firefox

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#34] [Announcements]
Addon version: v8.2.0.18a1

Now when we encounter VP9 content with non Opus/Vorbis audio, MKV format is selected instead of WebM. Fixing some issues on youtube and UHD content.

**Comment by bingalls** [comment]
I upgraded to 8.0.23 on a m1 mac, and Avast threw a virus warning. vdhcoapp is infected with MacOS:Agent-AIU [Tool]
<img width="605" alt="Screenshot 2024-02-16 at 11 19 54 AM" src="https://github.com/aclap-dev/video-downloadhelper/assets/761187/5734d6b0-3984-4a6b-803c-45e5321bf1a3">
Please verify!

**Reply by paulrouget** [reply]
[This is a false positive.](https://github.com/aclap-dev/video-downloadhelper/discussions/103), anybody can verify this, the code is Open Source: https://github.com/aclap-dev/vdhcoapp/ - Avast is flagging the CoApp wrongly.

Also, Agent-AIU is a Windows thing. You're on Mac.

And if anyone is worried, the app is verified by apple. Here is how you can verify:

- open a terminal
- run: `codesign -d -vvv /Applications/net.downloadhelper.coapp.app`

You'll see that the app has been verified by Apple:

```
Authority=Developer ID Application: ACLAP (4YP9AW3WW3)
Authority=Developer ID Certification Authority
Authority=Apple Root CA
```
